docs/apis/openpaasApi.md

```
@@ -347,10 +347,33 @@ PUT /api/configurations?scope=user
 ```
 
-Allows to update configuration of a user
+Allows to update configuration of a user.
+
+This sets the whole configuration object.
 
 Status code: 201
 
+### PATCH /api/configurations
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)